### PR TITLE
All feeds - updated fetch default param to true

### PR DESCRIPTION
**Pull request by orhovy**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27176

## Description
all feeds fetch default value is True

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 4.5.0
- [ ] 5.0.0
- [ ] 5.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation 

## Demisto Partner?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**

### CODE REVIEW

**Comment by amshamah419** [comment]
@bakatzir That sounds great.

File: Packs/Sixgill-Darkfeed/pack_metadata.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
 {
-    "name": "Darkfeed™ - Current Customer Edition",
```
Comment (bakatzir) [inline Packs/Sixgill-Darkfeed/pack_metadata.json:2]: please make sure not to change this line

File: Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml [hunk 2]
```diff
@@ -93,7 +94,7 @@ script:
     description: Retrieves a limited number of the indicators.
     execution: false
     name: unit42-get-indicators
-  dockerimage: demisto/taxii2:1.0.0.9904
+  dockerimage: demisto/taxii2:1.0.0.10275
```
Comment (bakatzir) [inline Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml:97]: please revert this as it causes an 400 error, same issue as in mitre feed.

File: Packs/Sixgill-Darkfeed/pack_metadata.json (hunk 1 above)
Comment (orhovy) [reply Packs/Sixgill-Darkfeed/pack_metadata.json:2]: the update-release-notes changes it from TM to it unicode. But I should have checked my commit. Will do next time.

File: Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml (hunk 2 above)
Comment (orhovy) [reply Packs/FeedUnit42/Integrations/FeedUnit42/FeedUnit42.yml]: Reverted

**Comment by orhovy** [comment]
Reverted the docker image but still the testplaybook is failing on error 400,
As the last pr the change of true to 'true' should not be responsible for the fail

**Comment by bakatzir** [comment]
@orhovy i thinks it needs to be `9219`, I just changed it now.

**Comment by bakatzir** [comment]
all versions passed, failed on dockerimage for unit42 and mitre, as known in the issue.

**Pull request merged**